src/scraper.py

```diff
@@ -142,10 +143,10 @@ def linkedin_scraper(self, max_jobs=25):
                     By.CLASS_NAME, 'jobs-search__results-list').find_elements(
                     By.CLASS_NAME, 'job-search-card')
 
-                # if len(job_list) > max_jobs:
```

Comment:
The commented parts can be removed

Reply:
what value do we want to set for max_jobs? 25 seems too low.